### PR TITLE
Replaces reference to public fields on `JSError`

### PR DESCRIPTION
**Pull request by nreid260**
* Motivation for features / changes
The Compiler is going to eliminate these fields.

* Technical description of changes
NO-OP

* Screenshots of UI changes

* Detailed steps to verify changes work correctly (as executed by you)

* Alternate designs / implementations considered

### CODE REVIEW

**Comment by nreid260** [comment]
Upcoming change in CL/268568571

**Comment by stephanwlee** [comment]
Thanks for the pointer. Will merge when CI passes.

**Comment by nreid260** [comment]
It looks like TB isn't building against the latest JSCopmpiler release. Any idea when the version will be incremented?

**Comment by nreid260** [comment]
Poke.

File: tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java [hunk 1]
```diff
@@ -515,15 +515,15 @@ public CheckLevel level(JSError error) {
             if (IGNORE_PATHS_PATTERN.matcher(error.sourceName).matches()) {
```
[the remark this answers is not in the document]
Comment (nreid260) [reply tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java]: Yes that should also be updated.

**Comment by nreid260** [comment]
rules_closure has already been updated.

**Comment by stephanwlee** [comment]
Thanks Nick. Btw, are you referring to https://github.com/bazelbuild/rules_closure/pull/417? 

@psybuzz we can use the commit hash since they don't seem to have a formal release with the change.

**Comment by nreid260** [comment]
Yes bazelbuild/rules_closure#417 advances the compiler version sufficiently.

File: tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (nreid260) [reply tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java]: Do you know how to do that? I'm not a git developer and this sidetrip has already cost me hours.

**Pull request closed without merging**